apis/apps/v1alpha1/cloneset_types.go

```diff
@@ -196,6 +196,8 @@ type CloneSetStatus struct {
 	CollisionCount *int32 `json:"collisionCount,omitempty"`
 
 	// Conditions represents the latest available observations of a CloneSet's current state.
+	// +patchMergeKey=type
+	// +patchStrategy=merge
```

Comment:
please add VolumeClaimTemplates patch merge comments.

Reply:
It seems that we cannot use the `name` field as the key for merging because name is a subfield of the `metadata` field